ff/serving/metadata/metadata_test.go

```diff
@@ -0,0 +1,1191 @@
+	m.recvToServer <- req
+	return nil
+}
+func (m *SourceStreamMock) RecvToClient() (*pb.Source, error) {
+	response, more := <-m.sentFromServer
+	if !more {
+		return nil, errors.New("no more to recieve")
+	}
+	return response, nil
+}
+
+func initializeCreate() (*MetadataServer, error) {
+	sugar := zap.NewExample().Sugar()
+	server := Config{
+		Logger: sugar,
+	}
+	metadataServer, err := NewMetadataServer(&server)
+	if err != nil {
+		return nil, err
+	}
+	if _, err := metadataServer.CreateUser(context.Background(), &pb.User{Name: "Simba Khadder"}); err != nil {
+		return nil, err
+	}
+	if _, err := metadataServer.CreateProvider(context.Background(), &pb.Provider{
+		Name:        "demo-s3",
+		Description: "local S3 deployment",
+		Type:        "Batch",
+		Software:    "BigQuery",
+		Team:        "",
+	}); err != nil {
+		return nil, err
+	}
+	if _, err := metadataServer.CreateEntity(context.Background(), &pb.Entity{
+		Name:        "user",
+		Description: "user description",
+	}); err != nil {
+		return nil, err
+	}
+	if _, err := metadataServer.CreateSourceVariant(context.Background(), &pb.SourceVariant{
+		Name:        "Transactions",
+		Variant:     "default",
+		Description: "Source of user transactions",
+		Type:        "CSV",
+		Owner:       "Simba Khadder",
+		Provider:    "demo-s3",
+	}); err != nil {
+		return nil, err
+	}
+	if _, err := metadataServer.CreateLabelVariant(context.Background(), &pb.LabelVariant{
+		Name:        "is_fraud",
+		Variant:     "default",
+		Description: "if a transaction is fraud",
+		Type:        "boolean",
+		Source:      &pb.NameVariant{Name: "Transactions", Variant: "default"},
+		Entity:      "user",
+		Owner:       "Simba Khadder",
+		Provider:    "demo-s3",
+	}); err != nil {
+		return nil, err
+	}
+	if _, err := metadataServer.CreateFeatureVariant(context.Background(), &pb.FeatureVariant{
+		Name:        "user_credit_score",
+		Variant:     "default",
+		Source:      &pb.NameVariant{Name: "Transactions", Variant: "default"},
+		Type:        "int",
+		Entity:      "user",
+		Owner:       "Simba Khadder",
+		Description: "User's credit score",
+		Provider:    "demo-s3",
+	}); err != nil {
+		return nil, err
+	}
+	if _, err := metadataServer.CreateModel(context.Background(), &pb.Model{
+		Name:        "user_fraud_random_forest",
+		Description: "Classifier on whether user commited fraud",
+	}); err != nil {
+		return nil, err
+	}
+	if _, err := metadataServer.CreateTrainingSetVariant(context.Background(), &pb.TrainingSetVariant{
+		Name:        "is_fraud",
+		Variant:     "default",
+		Description: "if a transaction is fraud",
+		Owner:       "Simba Khadder",
+		Provider:    "demo-s3",
+		Label:       &pb.NameVariant{Name: "is_fraud", Variant: "default"},
+		Features:    []*pb.NameVariant{{Name: "user_credit_score", Variant: "default"}},
+	}); err != nil {
+		return nil, err
+	}
+	return metadataServer, nil
+}
+
+func TestGetListUserMethods(t *testing.T) {
+	metadataServer, err := initializeCreate()
```

Comment:
@saadhvi27 do you know if you can silence this call: "{"level":"debug","msg":"Creating new metadata server"}" so it doesnt show in the test output constantly?